dandi/register.py

```diff
@@ -35,15 +49,13 @@ def register(name, description, dandiset_path=None, dandi_instance="dandi"):
 
     url = routes.dandiset_draft.format(**locals())
 
-    lgr.info(f"Registered dandiset at {url}. Please visit and adjust metadata.")
+    lgr.info(
+        f"Registered dandiset {dandiset['identifier']} at {url}. Please visit and adjust metadata."
+    )
+
     if dandiset_path:
+        lgr.info(f"Adjusting {dandiset_path} with obtained metadata")
         ds = Dandiset(dandiset_path, allow_empty=True)
         ds.update_metadata(dandiset)
-        return None
-    else:
```

Comment:
I feel like the function should at least warn if no dandiset path is provided or detected.

Reply:
Fair enough.  I have returned `.info` message in 7aa6877653c083ab53f9ecb865ba868734d14414 but in a location in the code above.  Since here we could get within an existing dandiset but which already has an `identifier` which we refuse above to modify (and thus do not set `dandiset_path`), so it would not be a matter of "not provided or detected", so this location would be suboptimal.